tests/azure/availset_test.go

```diff
@@ -19,33 +19,41 @@ func testAvailSetCreate(t *testing.T, f fixture.F) {
 	nash.Run(
 		f.Ctx,
 		t,
+		f.Logger,
 		"./testdata/create_avail_set.sh",
 		f.ResGroupName,
 		availset,
 		f.Location,
 	)
-	availSets := azure.NewAvailSet(f.Ctx, t, f.Session, f.ResGroupName)
+	availSets := azure.NewAvailSet(f.Ctx, t, f.Session, f.Logger, f.ResGroupName)
 	availSets.AssertExists(t, availset)
 }
 
 func testAvailSetDelete(t *testing.T, f fixture.F) {
-
 	availset := genAvailSetName()
 	nash.Run(
```

Comment:
@tiago4orion I'm not getting your proposal here, nash runner should just run nash scripts...but I'm creating a "test" data ? And the test data has a lot of stuff that does not make sense to run the script, I'm not even seeing where is the script that will be executed (path). Sorry, perhaps I'm missing something obvious.